src/pages/LoginPage/LoginPage.jsx

```diff
@@ -33,23 +37,53 @@ const LoginPage = () => {
           className='input-box'
           id='password'
           value={loginUser.password}
-          onChange={(e) =>
-            setLoginUser((prev) => ({ ...prev, password: e.target.value }))
-          }
+          onChange={(e) => {
+            setError("noerror");
+            setLoginUser((prev) => ({ ...prev, password: e.target.value }));
+          }}
+          onFocus={() => setError("noerror")}
+          required
         />
         <div className='forgot-psswrd'>
           <label>
-            <input type='checkbox' name='checkbox' className='checkbox-size' />
+            <input
+              type='checkbox'
+              name='checkbox'
+              className='checkbox-size'
+              defaultChecked
+            />
             Remember Me
           </label>
-          <p className='red-color forgot-psswrd-margin'>Forgot Password?</p>
+
+          <p
+            onClick={() => {
+              setLoginUser((prev) => ({
+                ...prev,
+                email: "adarshbalika@gmail.com",
+              }));
+              setLoginUser((prev) => ({ ...prev, password: "adarshbalika" }));
+            }}
+            className='test-login forgot-psswrd-margin'
+          >
+            Use Test Credentials
+          </p>
         </div>
 
         <button
+          type='submit'
           className='button login-button'
-          onClick={(e) => loginHandler(e, email, password)}
+          onClick={(e) => {
+            e.preventDefault();
+
+            if (email != "" || password != "") {
```

Comment:
**Suggestion:** Instead of checking whether email or password is empty or not, add required attribute in the input tag of email and password